src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -548,14 +551,21 @@ private List<String> identifyWinners(
       // * If this is a single-winner election in which it's possible for no candidate to reach the
       //   threshold (i.e. "first round determines threshold" is set), the tiebreaker will choose
       //   the only winner.
```

Comment:
I think this comment is a bit out of place now.

Reply:
Is it? The logic is the same on the next line.